### ISSUE TITLE
crash after update

### ISSUE THREAD

**Issue by lunchboxxx19** [iopleke/AlchemyPlusPlus#35]
updated to the latest builds off the jenkins.jakimfett website and got this crash

[Crashlog](https://www.dropbox.com/s/9uh4anpmjaaxs1u/alchemy%2B%2B%20crash.txt?dl=0)

**Comment by way2muchnoise** [comment]
You also need to install jakimbox (it is the lib APP works with)

**Comment by lunchboxxx19** [comment]
yup have that as well

**Comment by way2muchnoise** [comment]
Well I guess it isn't the proper version? At this point you'll need @jakimfett to show up tho.

**Comment by jakimfett** [comment]
Yeah, the latest builds are broken. Please use the download from CurseForge.

When I'm no longer on hiatus (I should know a bit more after my operation on Monday), my priority is to update JakimBox and Werkbench first, then AlchemyPlusPlus, then start poking Minechem again.

**Comment by Blue64** [comment]
`Version Checker Mod` referenced the most up-to-date version as `1.3.0.134` for `MineCraft 1.7.10`, would said version be the most recent `Stable Build`, or would said `Stable Build` be the current `CurseForge Alpha Build 1.2.0.89`?

I do hope you are doing well @jakimfett but is there perhaps any chance you would desire to perhaps at least do BugFixes on your current releases? Considering nobody has stepped forward to maintain it, that would still make You the most qualified apparently (as strange as that might feel), or perhaps you could at least search for someone to maintain your Mods in your (probably indefinite) absence. `:)`

Your [`Curse` Page](https://mods.curse.com/mc-mods/minecraft/alchemyplusplus#other-downloads) doesn't even mention your `1.7.10` builds, unlike your [`CurseForge` Page](https://minecraft.curseforge.com/projects/alchemyplusplus/files) which says your most recent `1.7.10` Release is `Alpha Build 1.2.0.89`. I personally loved your amazing work on your Mods, I probably actually have most of them (currently experiencing a [Crash Bug](https://github.com/Blue64/Infernal_Rift/issues/13) with [MineChem](https://github.com/iopleke/Minechem/issues/843))
